### ISSUE TITLE
Callout: High contrast issues

### ISSUE THREAD

**Issue by mikewheaton** [OfficeDev/office-ui-fabric-js#55]
_From @mikewheaton on June 5, 2016 19:24_

We have two reported high contrast issues for the Callout:
1. There are no outlines around the callout:
   - ![image](https://cloud.githubusercontent.com/assets/1382445/15807656/5a205712-2b18-11e6-8be7-50d539700b9e.png)
2. Callout links are purple in HC Black. They should be yellow.
   - ![image](https://cloud.githubusercontent.com/assets/1382445/15807660/72423b76-2b18-11e6-9fba-3572d15bd70e.png)

_Copied from original issue: OfficeDev/office-ui-fabric-core#630_

**Comment by Linda-Editor** [comment]
Microsoft no longer supports this content and will not be responding to bugs or issues. We recommend that you use the newer version, [Office UI Fabric](https://developer.microsoft.com/fabric), with React as your front-end framework. We are closing this issue; if you still need assistance with Fabric.js, visit [Stack Overflow/office-ui-fabric](https://stackoverflow.com/questions/tagged/office-ui-fabric).
